### PR TITLE
Add new coalesced test tab

### PR DESCRIPTION
**Pull request by dzickusrh**
As the work for coalesced testing results ramp up, create a pointer
to the data for early feedback.   The change points to the internal
grafana server for now.  Can be changed later.

Signed-off-by: Don Zickus <dzickus@redhat.com>

### CODE REVIEW

**Comment by dzickusrh** [comment]
Untested as I couldn't get the server running on my local laptop.
Happy to take suggestions for 'tab' name instead of coalesced-tests.

**Comment by alexandrasp** [comment]
> Untested as I couldn't get the server running on my local laptop.
> Happy to take suggestions for 'tab' name instead of coalesced-tests.

Sure, I will test your package in my local soon. And probably @gctucker and @khilman will have good tips and advices about it.

**Comment by dzickusrh** [comment]
Thanks @alexandrasp!  I forgot to mention, the goal of this change was to point to 
https://staging.kernelci.org:3000/.  I am not sure if I coded up the html correctly for that.

I should update the merge request with that info.

**Comment by khilman** [comment]
I tested this, and it appends a trailing slash before the `:3000`

**Comment by khilman** [comment]
Also, I think it should just be called "Experimental" or "WIP" instead of "coalesced" since I'm not really sure what coalesced means in this context (e.g. the existing UI is already coalescing tests from multiple labs.)

**Review by alexandrasp — COMMENTED** [review]
@dzickusrh I also tested it locally and it is not working. It is redirecting to: [https://staging.kernelci.org/3000](https://staging.kernelci.org/3000 )

File: app/dashboard/templates/base.html [hunk 1]
```diff
@@ -79,6 +79,12 @@
                             <i class="fa fa-stethoscope"></i>&nbsp;Tests<small>&nbsp;<sup>&#946;</sup></small>
                         </a>
                     </li>
+                    <li id="li-co-test">
+                        <a id="co-tests-l" href=":3000"
```
Comment (alexandrasp) [inline app/dashboard/templates/base.html]: will we use same server with different port? If yes is the answer, you need to be a little bit more explicit here..`href` needs server ip/address and probably there are some approaches that could be used to solve the problem.

Comment (gctucker) [reply app/dashboard/templates/base.html]: No it won't apply to production, so we need to have `https://staging.kernelci.org:3000`.

**Review by gctucker — COMMENTED** [review]
The full URL needs to be provided as the Grafana dashboard is only hosted on staging.kernelci.org.  This PR should be live a bit later today (4pm GMT).

**Comment by dzickusrh** [comment]
@khilman  @gctucker @alexandrasp - Updated with your feedback.  Thanks!

Switched to using 'WIP Test Reports' for now based on @khilman's suggestion.  Not sure it captures the effort clearly, but it is something for now.  Thanks!

**Review by alexandrasp — APPROVED** [review]
LGTM

**Comment by gctucker** [comment]
TBH I think `kcidb` as the text to show in the tab would have worked too, the stethoscope icon already shows it's test results.  But that's all very subjective of course...

**Comment by dzickusrh** [comment]
@gctucker - interesting point.  I think kcidb makes sense.  I guess if we start advertising that DB, it becomes obvious to folks where to find those results and what that tab is.
I know we are bikeshedding, but if noone else has an opinion either way by tomorrow, I will go ahead and modify the text to say 'kcidb'??

**Comment by khilman** [comment]
I think "kcidb" is a better name also.

A couple other minor things:

1) It should have the "beta" icon (like the current Tests tab)
2) it should open in a new window since there's no way to go back to the original after clicking around for awhile

**Comment by dzickusrh** [comment]
@khilman @gctucker - updated with your feedback.  Hopefully the 'new' tab thing works, untested.  I used KCIDB results as phrasing.  Not sure if that group of words makes sense or not.  Now I am thinking I should have used 'reports'. Thoughts?

**Comment by khilman** [comment]
Tested on local install. Beta icon looks good, and it opens in a new window.

**Comment by gctucker** [comment]
Yep, it's also visible on https://staging.kernelci.org/.

@dzickusrh If you're not planning to make further changes, please remove `WIP` from the PR subject and we'll get this merged.  It will then become live on kernelci.org during the next production update, most probably tomorrow (Friday).

File: app/dashboard/templates/base.html [hunk 2]
```diff
@@ -79,6 +79,12 @@
                             <i class="fa fa-stethoscope"></i>&nbsp;Tests<small>&nbsp;<sup>&#946;</sup></small>
                         </a>
                     </li>
+                    <li id="li-li-kcidb">
+                        <a id="kcidb-l" href="https://staging.kernelci.org:3000" target="_blank"
+                            title="View the KCIDB results">
```
Comment (patersonc) [inline app/dashboard/templates/base.html]: I don't feel strongly about this, but the "Tests" button also adds "(beta)" to the text you get when you hover over the button.

```suggestion
                            title="View the KCIDB results (beta)">
```

Comment (dzickusrh) [reply app/dashboard/templates/base.html]: Done.

**Comment by dzickusrh** [comment]
@patersonc - added (Beta) change.
@gctucker - removed WIP. I believe I am done.

Thanks everyone!

**Comment by patersonc** [comment]
Happy days.

**Pull request merged**